environment/src/main/java/com/sequenceiq/environment/environment/v1/converter/NetworkRequestToDtoConverter.java

```diff
@@ -31,6 +33,9 @@ public class NetworkRequestToDtoConverter {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(NetworkRequestToDtoConverter.class);
 
+    @Value("${cb.multiaz.azure.availabililtyZones:1,2,3}")
+    private Set<String> azureAvailabilityZones;
```

Comment:
is it always 3 for each region? As this is a default value, if I got it correctly.

Reply:
Yes. It's anyways getting filtered further based on instance type for instance group so even if there is a case where not all AZs are supported for a region, instance based filtering will automatically remove it.